docs/using-connect/integration.mdx

```diff
@@ -0,0 +1,374 @@
+---
+description: Integrating Connect with Blockchain Applications
+title: Integration
+icon: swap-arrows
+---
+
+<Info>
+    Connect is business-licensed software under the Business Source License (BSL). The source code is viewable; however, please reach out to us if you are interested in integrating. We are limiting the number of partnerships we engage with in 2024. We apologize in advance if we reach capacity and are unable to accommodate new integrations.
+</Info>
+
+
+
+This document will guide you through integrating Connect in your application.
+
+## Requirements
+- Go 1.22+
+- Cosmos SDK v0.50+
+
+
+
+## Integrating Connect
+
+Integrating Connect into your Cosmos SDK application requires a few simple steps.
+
+<Steps>
+    <Step title="Add Keepers">
+        Add the `x/marketmap` and `x/oracle` keepers to your app.
+    </Step>
+    <Step title="Setup Oracle Client">
+        Set up the Oracle client the application will use to get prices from the Connect oracle.
+    </Step>
+    <Step title="Set ABCI Method">
+        Set the PreBlock ABCI method, which is responsible for aggregating prices and writing them to the application's state.
+    </Step>
+    <Step title="Configure Vote Extensions">
+        Configure vote extensions with compression and storage strategies.
+    </Step>
+</Steps>
+
+## Keepers
+
+First, add the [x/marketmap](https://github.com/skip-mev/slinky/blob/main/x/marketmap/README.md) and [x/oracle](https://github.com/skip-mev/slinky/tree/main/x/oracle) keepers to the Application.
```

Comment:
many apps do not use `depinject`.  I think we should also have a guide for the non-depinject app wiring

Reply:
opened https://linear.app/skip/issue/BLO-1579/integration-guide-for-non-depinject-users